src/cron-schedule.ts

```diff
@@ -0,0 +1,206 @@
+
+    return new class extends CronSchedule {
+      public readonly expressionString: string = `${minute} ${hour} ${day} ${month} ${weekDay}`;
+      public _bind() {
+        return new LiteralSchedule(this.expressionString);
+      }
+    };
+  }
+
+  /**
+   * Create a cron schedule which runs every hour
+   */
+  public static hourly(): CronSchedule {
+    const minute = 0;
+    const hour = '*';
+    const day = '*';
+    const month = '*';
+    const weekDay = '?';
+
+    return new class extends CronSchedule {
+      public readonly expressionString: string = `${minute} ${hour} ${day} ${month} ${weekDay}`;
+      public _bind() {
+        return new LiteralSchedule(this.expressionString);
+      }
+    };
+  }
+
+  /**
+   * Create a cron schedule which runs every day at midnight
+   */
+  public static daily(): CronSchedule {
+    const minute = 0;
+    const hour = 0;
+    const day = '*';
+    const month = '*';
+    const weekDay = '?';
+
+    return new class extends CronSchedule {
+      public readonly expressionString: string = `${minute} ${hour} ${day} ${month} ${weekDay}`;
+      public _bind() {
+        return new LiteralSchedule(this.expressionString);
+      }
+    };
+  }
+
+  /**
+   * Create a cron schedule which runs every week on Sunday
+   */
+  public static weekly(): CronSchedule {
+    const minute = 0;
+    const hour = 0;
+    const day = '?';
+    const month = '*';
+    const weekDay = 0;
+
+    return new class extends CronSchedule {
+      public readonly expressionString: string = `${minute} ${hour} ${day} ${month} ${weekDay}`;
+      public _bind() {
+        return new LiteralSchedule(this.expressionString);
+      }
+    };
+  }
+
+  /**
+   * Create a cron schedule which runs first day of every month
+   */
+  public static monthly(): CronSchedule {
+    const minute = 0;
+    const hour = 0;
+    const day = 1;
+    const month = '*';
+    const weekDay = '?';
+
+    return new class extends CronSchedule {
+      public readonly expressionString: string = `${minute} ${hour} ${day} ${month} ${weekDay}`;
+      public _bind() {
+        return new LiteralSchedule(this.expressionString);
+      }
+    };
+  }
+
+  /**
+   * Create a cron schedule which runs first day of January every year
+   */
+  public static annually(): CronSchedule {
+    const minute = 0;
+    const hour = 0;
+    const day = 1;
+    const month = 1;
+    const weekDay = '?';
+
+    return new class extends CronSchedule {
+      public readonly expressionString: string = `${minute} ${hour} ${day} ${month} ${weekDay}`;
+      public _bind() {
+        return new LiteralSchedule(this.expressionString);
+      }
+    };
+  }
+
+  /**
+   * Create a custom cron schedule from a set of cron fields
+   */
+  public static custom(options: CronOptions): CronSchedule {
+    if (options.weekDay !== undefined && options.day !== undefined) {
+      throw new Error('Cannot supply both \'day\' and \'weekDay\', use at most one');
```

Comment:
Why not? Again looking at [CrontabGuru](https://crontab.guru/#*_*_2_*_4), this seems to be legit...

Reply:
As mentioned in the previous comment, let me know which format should I follow. 
I think since K8s mentions the wiki link, then that probably would work. Let me see if I can find something in their code.